ixmp/reporting/utils.py

```diff
@@ -378,6 +381,14 @@ def data_for_quantity(ix_type, name, column, scenario, filters=None):
     except KeyError:
         # 'equ' are returned without units
         attrs = {}
+    except ValueError as e:
+        if 'mixed units' in e.args[0]:
+            # Discard mixed units
+            log.warn('{} discarded for {!r}'.format(e.args[0], name))
```

Comment:
@OFR-IIASA as requested, the name of the variables are given when mixed units are discarded. These are also promoted to the `WARNING` log level to be consistent with other warnings.